pages/Addons.md

```diff
@@ -67,6 +67,7 @@ Galactifun | Seggan, Mooy1, GallowsDove, ProfElements | <details><summary>expand
 FNAmplifications | FN_FAL113 | <details><summary>expand</summary>A four in one addon that includes my FN Solar Generator, Material Generator, Machinery and Power Xpansion Addons which are based from the addons made by the slimefun addon community.</details> | [Readme](https://github.com/FN-FAL113/FN-FAL-s-Amplifications/blob/main/README.md) | [GitHub](https://github.com/FN-FAL113/FN-FAL-s-Amplifications) | [Builds](https://thebusybiscuit.github.io/builds/FN-FAL113/FN-FAL-s-Amplifications/main/)
 ElementManipulation | lucasGithuber | <details><summary>expand</summary>A slimefun addon that adds periodic table elements and more machines, giving the player the capacity to create items using these elements.</details> | [Readme](https://github.com/lucasGithuber/Element-Manipulation/blob/master/README.md) | [GitHub](https://github.com/lucasGithuber/Element-Manipulation) | [Releases](https://github.com/lucasGithuber/Element-Manipulation/releases)
 Crystamae Historia | Sefiraat | <details><summary>expand</summary>Crystamae Historia is a large-scale magic addon that adds a brand new series of crafting mechanics allowing players to craft magical gadgets, spells and tools. Crysta has a deep progression system, massive amounts of lore and enough spells/gadgets to keep any player engaged for a long, long, time.</details> | [Wiki](https://github.com/Sefiraat/CrystamaeHistoria/wiki) | [GitHub](https://github.com/Sefiraat/CrystamaeHistoria) | [Builds](https://thebusybiscuit.github.io/builds/Sefiraat/CrystamaeHistoria/master/)
+DankTech2 | Sefiraat | <details><summary>expand</summary>DankTech is a plugin that attempts to recreate the Dank Storage Mod and /Dank/Null/ items from Modded. Features include mass volume storage backpacks with auto-voiding, building, easy unloading and more!</details> | [Wiki](https://github.com/Sefiraat/DankTech2/wiki) | [GitHub](https://github.com/Sefiraat/DankTech2) | [Builds](https://thebusybiscuit.github.io/builds/Sefiraat/DankTech2/master/)
```

Comment:
There currently is no wiki so the wiki link points to the main github
not something that needs to be fixed just pointing it out

Reply:
Yeah I need to get on the wiki at some point D:

Reply:
fair